### ISSUE TITLE
Current collabora container version - Can't print

### ISSUE THREAD

**Issue by sriccio** [nextcloud/all-in-one#2949]
### Steps to reproduce
1. Login into nextcloud
2. Open office document
3. Try to print it

### Expected behavior 
Should trigger the print windows

### Actual behavior 
Does nothing except errors (SCP) in the browser console 

### Host OS 
Debian 12

#### Nextcloud AIO version 
6.1.2

#### Current channel 
Stable

#### Other valuable info 
Hello,

I gave a try to the nextcloud-aio fullstack with mastercontainer at 6.2.1 and the aio collabora container installed seems to be 23.05.0.5.

With CODE 23.05.0.5.1, the print function doesn't work.

According to browser console, it seems related to SCP:

```
Refused to frame 'blob:https://cloud.local/f4465b87-b90c-464c-a93e-54c5191c98db' because it violates the following Content Security Policy directive: "frame-src 'self' https://rating.collaboraonline.com/Welcome/welcome.html https://rating.collaboraonline.com/Rate/feedback.html https://nextcloud.com/pricing".

(index):6577 crbug/1173575, non-JS module files deprecated.
(anonymous) @ (index):6577
bundle.js:22766 Uncaught DOMException: Blocked a frame with origin "https://cloud.local" from accessing a cross-origin frame.
    at NewClass._onIframeLoaded (https://cloud.eu/browser/b6cb979/bundle.js:22766:34)
chromewebdata/:1 Not allowed to load local resource: blob:https://cloud.local/f4465b87-b90c-464c-a93e-54c5191c98db
```
I would guess that "blob:" is needed and missing in the frame-src SCP policy, but I'm not that much expert for this, and I don't know where I could add this to test it.

Is there a way to upgrade the collabora container to 23.05.1.x ?

Note: After testing this on another instance (non-AIO) I noticed this seems to be related to 23.05.0.x. Therefore, also reported it here: https://github.com/CollaboraOnline/online/issues/6892

Kind regards

**Comment by lachermeierm** [comment]
I noticed this problem a few weeks ago.

What I can tell you for sure: The problem will be solved with the next NextCloud AIO update.

This update installs the new Collabora version, which fixes the printing problem.

We just have to be patient a little longer.

See also: https://github.com/CollaboraOnline/online/issues/6624

**Comment by sriccio** [comment]
Thanks for the feedback. Let's wait for the update. Until that I worked around it by dropping the SCP header in the responses on the front reverse proxy...

**Comment by szaimen** [comment]
This is now released with v6.3.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
